clustermesh-apiserver/clustermesh/testdata/ciliumidentitites.txtar

```diff
@@ -2,6 +2,9 @@
 
 hive/start
 
+# Add two namespaces
+k8s/add namespace-1.yaml namespace-2.yaml
```

Comment:
It appears that this test is affected by a couple of small issues, I've opened a fix in https://github.com/cilium/cilium/pull/43372. Please rebase this PR on top of it once merged, and adapt as necessary.